### ISSUE TITLE
extend this app to other cameras (Nex, A6xxx,A7)

### ISSUE THREAD

**Issue by lh8511** [obs1dium/BetterManual#1]
Wow! 
Congratulations and thank you for coding this nice APP with many useful features in very simple and intuitive way to operate! As I am shooting 99% in Manual mode and mostly with legacy lenses this app has indeed the most needed for a very nice experience in that regard...

I tried this app on a Nex6! The app runs and was able to even take pictures, however since it haven't touch screen, was not able to change most of the features... With some camera keys re-mapping (e.g. MENU Keys, FN keys and RIGHT-LEFT Keys) and some adjust on _swipetouch_ functions it may be possible to  work with cameras without touch-screen.
-Would you please consider to extend the app for other cameras?

**Comment by obs1dium** [comment]
I made the other features accessible for non-touchscreen cameras, usability won't be that great at the moment, though.

**Comment by lh8511** [comment]
Now everything can be selected... and usability isn´t that bad either. Thank you ;-)

Still have some issues (on nex-6, but maybe it is working on other models):
-  **upper dial** ("DIAL_1") status always return zero no matter which direction:
  `case ScalarInput.ISV_DIAL_1_CLOCKWISE:
          case ScalarInput.ISV_DIAL_1_COUNTERCW:
              return onUpperDialChanged(getDialStatus(ScalarInput.ISV_DIAL_1_STATUS) / 22);
  `
  In order to change values up&down I was need to code status values for clockwise and countercw..
- **Bracket** and **timelapse** after selected and when clicked causes the app crash, but yet did not have time to inspect what is the cause..

**Comment by obs1dium** [comment]
I added exception logging in the latest release.

**Comment by stuomas** [comment]
How is the noise reduction disabling implemented? I mean could it be possible to disable it globally and permanently in all modes? Thanks alot by the way for the app!

**Comment by obs1dium** [comment]
The app doesn't do anything fancy, the setting for long exposure noise reduction just happens to be exposed through Sony's API.

If there is a hidden global setting for that, it may be possible to toggle it with [OpenMemories-Tweak](https://github.com/ma1co/OpenMemories-Tweak), but somebody would have to do the research first.

**Comment by stuomas** [comment]
Ok. I'm just not sure how all this is connected together and thought that maybe BetterManual is actually modifying that said setting. 

I'm pretty sure there is a hidden setting for this, since A6000 has this setting in the menu and it's basically the same camera. Sony just chose to gimp the A5100 menu for no apparent reason. Too bad the A5100 is not so common, I doubt anyone with the required skills is interested in researching this further.

**Comment by ghost** [comment]
TLDR.

NEX-6 - doesn't work. Logged in just to notify, no complaints.

**Comment by nishin0n** [comment]
TLDR.

NEX-5R - doesn't work. Logged in just to notify, no complaints.
Thanks for developing this interesting application!

**Comment by jameshoo007** [comment]
> TLDR.
> 
> NEX-5R - doesn't work. Logged in just to notify, no complaints. Thanks for developing this interesting application!

Hi there, did you install it and run on  NEX 5r，i installed it and run, but can not exit it
